tools/SourceKit/CMakeLists.txt

```diff
@@ -84,6 +84,11 @@ function(add_sourcekit_default_compiler_flags target)
     ANALYZE_CODE_COVERAGE "${analyze_code_coverage}"
     RESULT_VAR_NAME link_flags)
 
+  if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
```

Comment:
I realize this has been merged already, but should we be considering other non-Darwin platforms here?